internal/oauth/oauth.go

```diff
@@ -0,0 +1,190 @@
+package oauth
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"path"
+	"strings"
+	"time"
+
+	"github.com/foundriesio/fioctl/internal/config"
+	"github.com/sirupsen/logrus"
+)
+
+const uri string = "https://app.foundries.io/oauth"
+
+// OAuth interface.
+type OAuth interface {
+	Get() int
+	HasRefreshToken() bool
+	IsExpired() (bool, error)
+	Refresh() int
+	post(string, url.Values) (*[]byte, error)
```

Comment:
`post` isn't needed here. Since its lower case its not exposed anyway.

Reply:
Can't remember why I wanted it there, but yes, lower-case declarations are not exposed.